### ISSUE TITLE
Some puzzles about the badging workflow

### ISSUE THREAD

**Issue by xiaoya-yaya** [badging/diversity-and-inclusion#5]
Hi, my name is xiaoya, a student from Shanghai, China. 
I’ve been hanging around in the CHAOSS community for a while and particularly in the D&I working group.  I have scanned through the whole badging project but felt a little bit puzzled since I missed out the early months’ progress.

Here is my understanding of this workflow by now to apply for a badge: take project badge as an example, first I need to append my own project name(maybe a link?) into the entry table and leave the badge column blank; second I finish the questionnaire you provide.  After all this, I need to wait for other people to review my PR as well as take an assessment for my badging level, is that correct?

Here I have two questions:
1. After my project/event badging being assessed, I assume the badge will be added into the entry table, and apart from this, will it show in other place, like it will be after my project name like a tail or something?
2. And are the events strictly follow the terms defined by GitHub?
https://developer.github.com/webhooks/

Anyway, look forward to see how the workflow is going to be detailed and wish I could involve myself more !

**Comment by bistaastha** [comment]
Hi @xiaoya-Esther !

> Here is my understanding of this workflow by now to apply for a badge: take project badge as an example, first I need to append my own project name(maybe a link?) into the entry table and leave the badge column blank; second I finish the questionnaire you provide. After all this, I need to wait for other people to review my PR as well as take an assessment for my badging level, is that correct?

I think you understand the current workflow correctly.

Addressing your first question, yes, the badges are meant to be showcased on places like a README or a website.

About your second question, are you asking about Event badging or just GitHub events?

Do let me know if anything was unclear from my side. And there are ways you can get more involved! You have already taken the first step. If there's anything you want to discuss, you can:

- Either open another issue here, or
- Hit us with a mail on the D&I mailing list

I remember that you have been a part of the D&I working group calls before, and that is also a great place to directly talk about your ideas with us :)

**Comment by xiaoya-yaya** [comment]
Thanks! That helps. I think now I know the whole procedure better.

> About your second question, are you asking about Event badging or just GitHub events?

Well, I mean about the Event badging, how you guys define "Event"? Are they follow the event types defined by the GitHub webhooks website?

Actually I'm thinking of applying for the GSoD ^_^ I have read the project ideas, guess before writing my proposal, I shall just get a further comprehension of the project and involve myself into more discussions!

**Comment by bistaastha** [comment]
When it comes to Event badging, we refer to events where people meet up, which would possibly include stuff like tech related conferences.

GitHub events are not related to Event badging.

@xiaoya-Esther It is great to see your enthusiasm towards the project idea:)

**Comment by xiaoya-yaya** [comment]
Get it, so the "Event" semantically refers to the actual meetings that occur in real world.

**Comment by Nebrethar** [comment]
Hi @xiaoya-Esther and welcome. I'm glad to see you here with such interest in the project 😁

The original focus we had in mind for events was similar to "tech get-togethers, conferences, festivals, and pretty much any other tech event". We certainly won't turn down a non-tech event if our framework will still applies. Our scope is widening to digital events, given the current atmosphere.

Regarding how the badges are reviewed, I think it would be helpful to refer to the [Journal of Open Source Software](https://joss.readthedocs.io/en/latest/submitting.html#the-review-process) under "review process".  This is where we are building out review process and a portion of our workflow from.

We are also modeling some methods for display and validation from the [Core Infrastructure Initiative](https://bestpractices.coreinfrastructure.org/en) badging program, though their work is mostly technical. The self-reporting nature of the submission process is a good idea of theirs that we took on.

Glad to answer any other questions 😄

**Comment by xiaoya-yaya** [comment]
Thanks @Nebrethar ! the JOSS and CII Badge Program you provided me is super helpful:)

**Comment by xiaoya-yaya** [comment]
I can see that the JOSS reviewers are assigned to review the original software repository and the CII is most likely to fill out a questionnaire (something like what we will perform but will be peer-reviewed by the details ) and I found during the filling process I can actually fake my project’s situation.

so another question aroused,  when a reviewer performs his/her review task, what exactly should he review? I mean, Is the detailed answers in the PR template the main thing to concern, or the actual project/event (whether a repository or an event website) should also be a compulsory task for the reviewer to check. Which part should a reviewer mainly focus on? 

It’s a question about whether it’s required and how to verify the authenticity of the answers. I think that’s something that needs to be addressed in the reviewer guidelines.


Plus, @bistaastha  I found your [project notes](https://docs.google.com/document/d/1zMh-DwCSpc3FjS7YXkdJ2PsTN0YLXvaTo-uXpBNNwTA/edit#) and [GSoC proposal](https://docs.google.com/document/d/1YPAlVUhUZG6Gc4k8l5zxZz6gWoHAG-UsLlGEOiQkvGE/edit?pli=1) is particularly helpful with my understanding of the whole procedure😄,  can I use it as an authority reference of the exact procedure that we are going to develop?

**Comment by Nebrethar** [comment]
Hi @xiaoya-Esther and I apologize for the late response. 😁 Regarding your first question...

The workflow is shifting piece by piece right now, but there are some things that probably won't change. The most important details as I understand them are:
- The submitter fills out the entire template, all questions. This is necessary for a project or event to be considered for a full review.
- Some items, such as the presence of a Code of Conduct, will have to be fact-checked by a reviewer. Items like this should be **easy** to find on the project or event website.
- The project will have some level of subjective review, especially because we have essay questions (provide a statement etc..). This needs to be taken into account when creating a system of docs that supports reliable and unbiased reviews

**In summary:** the reviewer is encouraged to focus on providing a reasonable, unbiased, and constructive review of the projects and events they work with. As for the details of the workflow itself, that is still under construction. 😅 

I am also going to include @bistaastha because that's an important question for her and I cannot answer it myself.

**Comment by Nebrethar** [comment]
Hi @xiaoya-Esther and I hope you are having a great day. I have good news.

I am leaving this here to notify you that our [microtasks are available](https://github.com/badging/meta/issues/25) for Season of Docs.

Thanks you!

**Comment by xiaoya-yaya** [comment]
Thanks a lot for the information @Nebrethar ! I will check this out and wish you a wonderful day, too. 
Best
